system/Email/Email.php

```diff
@@ -2358,12 +2358,16 @@ protected function getSMTPData()
      */
     protected function getHostname()
     {
-        if (isset($_SERVER['SERVER_NAME']))
+        if ($serverName = $_SERVER['SERVER_NAME'] ?? null)
         {
-            return $_SERVER['SERVER_NAME'];
+            return $serverName;
         }
```

Comment:
This one also is a logic change from anything in `SERVER_NAME` that was falsy. For example:
```
$_SERVER['SERVER_NAME'] = '';

// Previously
echo $this->getHostname(); // ''

// Now
echo $this->getHostname(); // '[127.0.0.1]'
```

Reply:
Is this good or bad? I would argue that yes, it's a logic change, but an empty host name would definitely be invalid and I assume unwanted 🤔 - what should I do here?

Reply:
Let's keep the logic for now, since we're really focused on styles for this release. I will open an Issue so we come back and take a look at what the expected and desired output form this is. FWIW I totally agree that the current logic is not ideal.

Reply:
Actually, when I went to make the note I noticed this is in `Email`, which is being replaced soon anyway, so I don't really care either way :P

Reply:
I changed it back to `isset()` but rewrote it slightly to avoid the ternary operator for readability (it does more than just `$a ? $a : $b`),